afa/src/main/java/com/fox/platform/training/serv/RestTrainingVerticle.java

```diff
@@ -0,0 +1,47 @@
+package com.fox.platform.training.serv;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.handler.BodyHandler;
+
+public class RestTrainingVerticle extends AbstractVerticle{
+	
+	  @Override
+	    public void start() {
+		  
+		  Router router = Router.router(vertx);
+		  router.route().handler(BodyHandler.create());
+		  router.get("/channels/:country").handler(context -> processRequest(context));

```

Comment:
Sería mejor utilizar algo como:

`router.get("/channels/:country").handler(this::processRequest);`